DOM/to-do-app/JS/taskHelpers.js

```diff
@@ -54,6 +54,10 @@ function showAllTasks() {
         if (filterStatus === "all" || !filterStatus) return true;
         return task.status === filterStatus;
       })
+      .filter((task) => {
+        if (!filterByTitle) return true;
+        return task.title.toLowerCase().includes(filterByTitle);
```

Comment:
what if `filterByTitle` will be with capitalized letters?
use `toLowerCase` on both values